### DISCUSSION TITLE
Pitney Bowes Account

### DISCUSSION THREAD

**Discussion by dgethers** [msusdev/Innovation-Challenge#8] [Q&A]
Hi folks,

As we develop our project for the Pitney Bowes executive challenge. We were wondering if there was a test account that we could use if we wanted to call the api for information in the sandbox account?

Otherwise we will mock up the interface.

**Comment by MatthewCalder-msft** [comment]
Great question. Let me reach out to the team at Pitney Bowes to get their guidance. I was asked yesterday if anyone working on that challenge would want something like an "office hours" or other way to connect directly with them. Let me know if that would be interesting to you.

**Reply by dgethers** [reply]
Office hours would definitely be interesting to me.

**Reply by MatthewCalder-msft** [reply]
We're double checking, but we think the sandbox environment should give you the access you need for testing. Also looking to confirm an office hours session on Thursday. Feel free to reach out to me directly on email too macalde@microsoft.com

**Reply by hari70** [reply]
We are getting credentials for 7 sandbox accounts. will distribute them shortly.

**Comment by MatthewCalder-msft** [comment]
We'll have a subject matter expert from Pitney Bowes available for an office hours session on Thursday, June 6 at 12:30 pm Eastern / 9:30 am Pacific. [Here is a link to the Teams meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_ODkwZDViOWEtMjBkNC00MGZhLWEwZTEtNGE0ZjIxNTNmMGU2%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%2220ae62c4-ebe4-4112-b407-9bfbf066992e%22%7d).

**Reply by CarnegieJ** [reply]
You read my mind. :-)  Thank you.

**Comment by MatthewCalder-msft** [comment]
OK, I have the info from Pitney Bowes that would be needed to set up the sandbox environment. Mail me directly macalde@microsoft.com
